extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/discovery/BridgeDiscoveryService.java

```diff
@@ -39,6 +41,7 @@
  * @author Michael Ochel - Initial contribution
  * @author Matthias Siegele - Initial contribution
  */
+@Component(service = DiscoveryService.class, immediate = true, configurationPid = "discovery.digitalstrom")
```

Comment:
The `activate()` and `modified()` methods also need to have `@Activate` and `@Modified` annotations

Especially for the `modified()` method this is crucial, as unlike `activate()`/`deactivate()` it will _not_ be called by naming convention implicitly!

Reply:
I've added the missing annotations @sjka.

Should we also fix this for all other classes extending the `AbstractDiscoveryService`? It would fix the issue that background discovery configuration changes are not immediately applied.

Reply:
That would be awesome! Sorry, I noticed that too late... One the positive side: Now you won't need to amend bazillions of PRs but can do it in one.